migrations/versions/4d93784a25e5_add_authz_column.py

```diff
@@ -38,8 +38,10 @@ def upgrade():
 
     # extract existing PK (guid) and authz data (resource_path) from the metadata column
     connection = op.get_bind()
-    results = connection.execute("SELECT guid, data from metadata").fetchall()
-    for r in results:
+    query = "SELECT guid, data from metadata"
+    results = connection.execute(query)
+    r = results.fetchone()
+    while r:
         guid, data = r[0], r[1]
         # default values for authz (["/open"])
         authz_data = json.loads(DEFAULT_AUTHZ_STR)
```

Comment:
let's take this line out of the loop

Reply:
should the similar code in the downgrade also be removed? 
Like this `if` statement
`if authz_data.get(authz_key) != default_paths:`